src/cli/programutilities.h

```diff
@@ -8,4 +8,6 @@ namespace Ripes {
 QString loadFlatBinaryFile(Program &program, const QString &filepath,
                            unsigned long entryPoint, unsigned long loadAt);
 
+bool loadElfFile(Program &program, QFile &file);
```

Comment:
Nit: now that you're technically re-declaring the function, i'd appreciate a comment describing the function (I'm sneakily trying to sneak in more documentation in the codebase 😉).